plugin/src/test/java/org/opensearch/ml/rest/RestMLRemoteInferenceIT.java

```diff
@@ -237,6 +238,39 @@ public void testPredictRemoteModel() throws IOException, InterruptedException {
         assertFalse(((String) responseMap.get("text")).isEmpty());
     }
 
+    public void testPredictWithAutoDeployAndTTL_RemoteModel() throws IOException, InterruptedException {
```

Comment:
Can't we use any existing test? So that we don't need any extra api call again? Also waiting for 71 seconds!!!!

May be use one test and keep the model Id public so that at the end of this whole IT test, we can see if that model is already un deployed or not for not being used?

Reply:
It's in my plan to remove some other duplicate tests to save time.